tpu/src/test/java/tpu/TpuVmIT.java

```diff
@@ -210,4 +210,27 @@ public void testStopTpuVm() throws IOException, ExecutionException, InterruptedE
       assertEquals(returnedNode, mockNode);
     }
   }
+
+  @Test
+  public void testCreateTpuVmWithStartupScript() throws Exception {
+    try (MockedStatic<TpuClient> mockedTpuClient = mockStatic(TpuClient.class)) {
+      Node mockNode = mock(Node.class);
```

Comment:
nit: I do not think that mocking this class is required. I am unaware about methods of this class that perform calls to backend. I would suggest to mock the implementation that supposedly populates the returned value with proper data and assert this.

Reply:
Thank you! Fixed